scripts/trigger-mirror.py

```
@@ -29,18 +30,34 @@ def gitlab_mirror_api(project_id: str, method: str = "GET"):
         with urllib.request.urlopen(req) as response:
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)